### PR TITLE
Iris onto master

### PR DESCRIPTION
**Pull request by hanssv**

### CODE REVIEW

**Review by velzevur — APPROVED** [review]

**Review by cytadela8 — APPROVED** [review]

**Comment by cytadela8** [comment]
(blocking to let me investigate)

File: serializations.md [hunk 1]
```diff
@@ -421,6 +421,8 @@ If the value is the empty binary the key is pruned from the tree.
 
 The content of the contract store depends on [the ABI and the VM version](/contracts/contract_vms.md).
 
+From FATE VM version 2 stored code is the same as from the coresponding contract create operation. FATE VM version 1 contracts are stored without init function. Only way to check init function of a FATE v1 deployed contract is to look at the Contract create transaction.
```
Comment (uwiger) [inline serializations.md:424]:
```suggestion
From FATE VM version 2, stored code is the same as from the corresponding contract create operation. FATE VM version 1 contracts are stored without an `init` function. The only way to check the `init` function of a FATE v1 deployed contract is to look at the Contract create transaction.
```

**Review by uwiger — APPROVED** [review]

**Review by cytadela8 — APPROVED** [review]
Found the "missing" paragraph in sophia repo. All good, I belive no changes where lost.

File: serializations.md (hunk 1 above)
Comment (cytadela8) [reply serializations.md:424]: We could use some CI with aspell :persevere:

**Comment by marc0olo** [comment]
is there any reason why this isn't merged yet?

**Pull request merged**